testing/xsel/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: dai9ah <dai9ah@protonmail.com>
+# Maintainer:
+pkgname=xsel
+pkgver=1.2.0
+pkgrel=0
+pkgdesc="Command-line program for manipulating the X selection"
+url="http://www.vergenet.net/~conrad/software/xsel"
+arch="all"
+license="custom"
+makedepends="autoconf automake libtool libxt-dev"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/kfish/$pkgname/archive/$pkgver.tar.gz"
```

Comment:
Please use official sources where available: the URL suggests http://www.vergenet.net/~conrad/software/xsel/download/xsel-1.2.0.tar.gz as an official source.  This should additionally allow you to use `./configure` instead of autotools, removing the dependencies on autoconf, automake, and libtool.